keyboards/pimoroni/keybow2040/mcuconf.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+/*
+ * implicitly LED driver chips
+ */
+#define DRIVER_COUNT 1
+
+/*
+ * I2C address when IS31FL3731 AD pin is connected to ground
+ */
+#define DRIVER_ADDR_1 0b1110100
+
```

Comment:
`DRIVER_COUNT` and `DRIVER_ADDR_1` are QMK configuration parameters; they should be set in `config.h`, not in `mcuconf.h`.
```suggestion
```

Reply:
Fixed in round three!